Gemfile

```diff
@@ -0,0 +1,14 @@
+source 'https://rubygems.org'
+
+gem 'sinatra'
+gem 'datamapper'
+gem 'bcryp'
+gem 'dm-validations'
+gem 'rack-flash3'
+
+group :test do
+  gem 'rspec'
+  gem 'capybara'
+  gem 'capybara/cucumber'
+  gem 'database_cleaner'
+end
```

Comment:
You need to check in your gemfile.lock http://viget.com/extend/bundler-best-practices